eng/verify-nupkgs.ps1

```diff
@@ -22,7 +22,7 @@ function Confirm-NugetPackages {
         "MSTest.Sdk"                                  = 15
         "MSTest.TestFramework"                        = 148
         "MSTest.TestAdapter"                          = 75
-        "MSTest"                                      = 6
```

Comment:
Extra `_._` in lib for:

- net6.0
- net7.0
- net8.0
- net9.0
- net462
- netcoreapp3.1
- netstandard2.0
- uap10.0

Total 8 TFMs. 6 (old files) + 8 (new files) = 14

Reply:
Do we need all net TFMs? Or shall we match the adapter ones?

Reply:
The ones I added are the adapter ones per https://www.nuget.org/packages/MSTest.TestAdapter/3.7.3#dependencies-body-tab